WaterLog_Backend/Procedures.cs

```diff
@@ -1065,6 +1066,31 @@ public List<MonitorHeat> getMonitorsFaultLevels()
             return daily;
 
         }
+
+        public async Task<List<TankObject>> getObjects()
+        {
+            var allGrouped  = await _db
+                            .TankReadings.Where(a => a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year )
```

Comment:
This line is too long here is a suggestion 
```
var allGrouped = await _db
	.TankReadings
        .Where(a => a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year )
	.GroupBy(b => b.TankMonitorsId)
	.ToListAsync();
```

Reply:
No anonymous variable, function programming like lambda function in a sense.
ill indent it better?

Reply:
ohhh sorry misunderstood you

Reply:
changed it